include/af/defines.h

```diff
@@ -63,20 +63,20 @@ typedef long long intl;
 typedef unsigned long long uintl;
 
 typedef enum {
-    AF_SUCCESS=0,
-    AF_ERR_INTERNAL,
-    AF_ERR_NOMEM,
-    AF_ERR_DRIVER,
-    AF_ERR_RUNTIME,
-    AF_ERR_INVALID_ARRAY,
-    AF_ERR_ARG,
-    AF_ERR_SIZE,
-    AF_ERR_DIFF_TYPE,
-    AF_ERR_NOT_SUPPORTED,
-    AF_ERR_NOT_CONFIGURED,
-    AF_ERR_INVALID_TYPE,
-    AF_ERR_INVALID_ARG,
-    AF_ERR_UNKNOWN
+    AF_SUCCESS            =   0,
+    AF_ERR_INTERNAL       =  -1,
+    AF_ERR_NOMEM          = 101,
```

Comment:
I was going to do that but left it in place because of consistency with the old errors. But I guess this would be the right time to change them.